test/blackbox-tests/test-cases/directory-targets/github6168.t/fakenpm.ml

```diff
@@ -0,0 +1,5 @@
+let () =
+  let _ = Sys.mkdir "fakenode_modules" 0o777  in
+  let _ = Sys.mkdir "fakenode_modules/foo" 0o777 in
+  let _ = Unix.symlink "file" "./fakenode_modules/foo/bar" in
```

Comment:
Why do you use this `let _` form? All these functions return `unit` and should just be terminated with semicolons.

Reply:
I had the `Sys.command` before which returned an `int` which is why I did that.